### DISCUSSION TITLE
Next Image and Google Places Photos

### DISCUSSION THREAD

**Discussion by Nate-gg** [vercel/community#563] [Help] [answered]
In my project I have a portion that displays and image either from Instagram or Google Places. Images are pulled from both places, stored in an array, randomized then 12 are shown. Locally all works fine, however when I deploy to Vercel the Google photos do not show. I have the domains in my config, if I use a normal <img> they work fine. It seems its something with <Image> and the Google Places photos not liking each other, but I can't for the life of me figure out why. 

The Google Places URL does redirect, the API serves a maps.googleapis.com URL which redirects to lh3.googleusercontent.com to display the image. Not sure if maybe that has something to do with the issue.

**Comment by swarnava** [comment, marked as answer]
Hi, Can you reach out to https://github.com/vercel/next.js/discussions? Thanks!

**Reply by Nate-gg** [reply]
Thanks, I posted the question there :)
